### PR TITLE
Created docker-compose configuration and tested in local to run as container

### PR DESCRIPTION
**Pull request by jinnabaalu**

### CODE REVIEW

**Comment by hkamran80** [comment]
Wouldn't it be better to use a `.env` file that is referenced across each Compose service?

**Comment by jinnabaalu** [comment]
> Wouldn't it be better to use a `.env` file that is referenced across each Compose service?

That is individual choice, rather complicating the compose, i want to keep it plain, when you run in your environment it should be based on their thought to apply.

**Review by dodwmd — APPROVED** [review]

File: config/environments/production.rb [hunk 1]
```diff
@@ -43,7 +43,7 @@
   # config.assume_ssl = true
 
   # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
-  config.force_ssl = true
+  config.force_ssl = ENV.fetch("SSL_ENABLE", "true") == "true"
```
Comment (aleonmon) [inline config/environments/production.rb:46]: I would suggest using either `SSL_ENABLED` or `ENABLE_SSL` and keeping it consistent for the upcoming env vars

Comment (jinnabaalu) [reply config/environments/production.rb:46]: You want me to have the variable `ENABLE_SSL`  confused. Please let me know I can update

Comment (aleonmon) [reply config/environments/production.rb:46]: `ENABLE_SSL` or `SSL_ENABLED` => OK

`SSL_ENABLE ` => NOT OK

**Comment by VDuda** [comment]
Tested this branch in my environment -  ```FATAL:  database "maybe" does not exist``` - I believe the database should be set to `maybe_production`? Correct me if I'm wrong, working through the issue now.

File: .github/workflows/create-docker-image-push-toghcr.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+name: Build and Push Docker Image
+
+on:
+  push:
+    branches:
+      - main
```
Comment (thomas-mc-work) [inline .github/workflows/create-docker-image-push-toghcr.yml:6]: Wouldn't it suffice to only build and push on creating tags?

**Comment by zachgoll** [comment]
Closing this out for now.  I think these changes belong in a dedicated PR to build out a Dockerfile, address SSL environment variables, and an automated push flow for the Dockerfile.  This is all being tracked in #295

**Pull request closed without merging**